_posts/2018-07-12-open-source-design-nyc.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: event
+title:  "Open Source Design NYC: July 12"
```

Comment:
Also the year would be nice to be in the title, since on the long run it would help us identify them easier. We should make this a standard for all the future events.